src/plugins/intel_gpu/src/plugin/ops/paged_attention.cpp

```diff
@@ -72,7 +72,19 @@ static void CreatePagedAttentionExtensionOp(ProgramBuilder& p, const std::shared
     std::shared_ptr<ov::op::v0::Constant> alibi_const = ov::as_type_ptr<ov::op::v0::Constant>(op->get_input_node_shared_ptr(alibi_idx));
     OPENVINO_ASSERT(alibi_const != nullptr);
     prim.has_alibi = ov::shape_size(alibi_const->get_output_shape(0)) > 0;
-    prim.has_rotated_blocks = op->get_input_size() == 16;
+
+
+    const size_t score_aggregation_idx = 13;
+
+    auto score_aggregation_input = ov::as_type_ptr<ov::op::v0::Parameter>(op->get_input_node_shared_ptr(score_aggregation_idx));
+
+    if (!score_aggregation_input) {
+        prim.has_score_aggregation = false;
+    } else {
+        prim.has_score_aggregation = !score_aggregation_input->get_output_partial_shape(0).is_dynamic();
```

Comment:
Shouldn't it be `prim.has_score_aggregation = score_aggregation_input->get_output_partial_shape(0).is_dynamic();` ?

Reply:
Actually I'm not sure, what would be the semantically right way here for determining missing input other than checking for nullptr - what other options would have to be considered?